### ISSUE TITLE
`noinspection` for custom inspections false positive

### ISSUE THREAD

**Issue by Simbiat** [kalessil/phpinspectionsea#1973]
| Subject        | Details                                                                       |
| :------------- | :---------------------------------------------------------------------------- |
| Plugin         |  Php Inspections (EA Ultimate), 2024.3 |
| Language level | e.g. PHP 8.4                        |

Current behaviour
--
I have custom inspections and when I use
```php
/** @noinspection NoMBMultibyteAlternative */
```
I get `[EA] This call can be merged with the previous.` warning. You can use [this profile](https://github.com/Simbiat/simbiat.ru/blob/master/.idea/inspectionProfiles/Project_Default.xml) for testing

Expected behaviour
--
No warnings

Environment details
---

PhpStorm 2024.3.4
Build #PS-243.25659.45, built on February 27, 2025
Licensed to Dmitrii Kustov
You have a perpetual fallback license for this version.
Subscription is active until March 3, 2026.
Runtime version: 21.0.6+8-b631.39 amd64 (JCEF 122.1.9)
VM: OpenJDK 64-Bit Server VM by JetBrains s.r.o.
Toolkit: sun.awt.windows.WToolkit
Windows 11.0
GC: G1 Young Generation, G1 Concurrent GC, G1 Old Generation
Memory: 5120M
Cores: 16
Registry:
  debugger.new.tool.window.layout=true
  run.processes.with.pty=TRUE
  ide.experimental.ui=true
  terminal.new.ui=true
Non-Bundled Plugins:
  com.intellij.ml.llm (243.23654.270.16)
  ru.adelf.idea.dotenv (2024.3)
  de.espend.idea.php.annotation (11.2.0)
  fr.adrienbrault.idea.symfony2plugin (2024.1.276)
  com.kalessil.phpStorm.phpInspectionsUltimate-mp (2024.3)

**Comment by ea-inspections-team** [comment]
Thank you for reporting @Simbiat ! The false-positive has been addressed.
